_install_tabs/4-other.md

```diff
@@ -0,0 +1,18 @@
+---
+tabId: other
+tabLabel: Other
+fallbackNoJS: true
+language: en
+---
+<div class="scala-in-action-content">
+  <div class="scala-in-action-code">
+    <div class="wrap">
+      <div class="scala-text scala-text-large">
+        <noscript>
+          <p><span style="font-style:italic;">JavaScript is disabled, showing the default options.</span></p>
+        </noscript>
+        <p>Follow the documentation from Coursier on <a href="https://get-coursier.io/docs/cli-installation" target="_blank">how to install and run <code class="bash">cs setup</code></a>.</p>
```

Comment:
I suggest making it clearer that users have to first install `cs`, and then run `cs setup`.

```suggestion
        <p>Follow the documentation from Coursier on <a href="https://get-coursier.io/docs/cli-installation" target="_blank">how to install <code>cs</code></a>, and then <a href="https://get-coursier.io/docs/cli-setup" target="_blank">run <code>cs setup</code></a>.</p>
```

What does `class="bash"` do? I see no changes when I remove it.

Reply:
hljs automatically injects its own classes into `<code>` inside of a `<pre>`, so I provide the language explicitly so it does not do that. Also, the instructions on the coursier website tell you to run `cs setup`, not only how to install `cs`

Reply:
OK for the highlighting, and ok for `cs setup`, you’re right!